modules/caesarcipher.js

```diff
@@ -0,0 +1,43 @@
+if (args.length < 1) {
+    ktaneSpeak("Not enough arguments.")
+    return;
+}
+
+var display = args[0];
+var offset = 0;
+var offsetLock = false;
+
+if (bombinfo.getPortsCount("parallel") > 0 && bombinfo.indicators.exists("nsa") && bombinfo.indicators.isLit("nsa")) {
```

Comment:
u can just do isLit coz if it is lit then it must exist

Reply:
but what if it doesn't exist

Reply:
then isLit is false